Automattic/wp-calypso
# Jetpack cloud: update sidebar items

#### Changes proposed in this Pull Request

This PR updates all icons in the Jetpack cloud sidebar navigation to the new icons we've been working on that are fully compatible with Gutenberg/Core's style.

It does so by adding a `JetpackIcons` components which ensures the markup for all icons is consistent across the board — not sure if this is the right approach though, would appreciate your thoughts on this.

* Adds refreshed icons to a new `JetpackIcons` component.
* Updates all the icons in cloud.jetpack.com.
* Aligns icons and labels.

#### Testing instructions

* Fire up this PR using `yarn start-jetpack-cloud`.
* Head up http://jetpack.cloud.localhost:3000/
* Visit both the Manage Sites and the Partner Portal sections.
* Ensure all icons look like the screenshots below, and everything is nicely aligned.

Related asana card: 1199980337313591-as-1200907868342144

#### Screenshots

Before | After
--|--
![image](https://user-images.githubusercontent.com/390760/131996115-5b206b49-14c0-4e10-8f4d-8ad20e36803f.png) | ![image](https://user-images.githubusercontent.com/390760/131996167-f7f139e8-7223-4c80-8032-37308a81d478.png)
![image](https://user-images.githubusercontent.com/390760/131995897-0a86de86-dda9-45ee-bfc0-1617440dbd51.png) | ![image](https://user-images.githubusercontent.com/390760/131995881-f92a132f-c43d-41f5-aada-05e4f98ff917.png)
![image](https://user-images.githubusercontent.com/390760/131995940-7f0df493-f759-46b8-b937-3a6b004ac9ef.png) | ![image](https://user-images.githubusercontent.com/390760/131995930-67393245-3a1b-4014-a095-70b27777767b.png)
